molgenis-swagger/src/main/resources/templates/view-swagger.ftl

```diff
@@ -0,0 +1,525 @@
+          description: For an unknown entity.
+          schema:
+            $ref: "#/definitions/ErrorMessageResponse"
+        401:
+          description: If the user lacks the proper permissions.
+          schema:
+            $ref: "#/definitions/ErrorMessageResponse"
+        500:
+          description: If a runtime exception occurs.
+          schema:
+            $ref: "#/definitions/ErrorMessageResponse"
+definitions:
+  CopyEntityRequest:
+    type: object
+    properties:
+      newEntityName:
+        type: string
+    required:
+      - newEntityName
+  LoginRequest:
+    type: object
+    properties:
+      username:
+        type: string
+      password:
+        type: string
+  LoginResponse:
+    type: object
+    properties:
+      token:
+        type: string
+      username:
+        type: string
+  EntityCollectionRequestV2:
+    type: object
+    properties:
+      q:
+        type: string
+      aggs:
+        type: string
+      sort:
+        type: string
+      attrs:
+        type: string
+      start:
+        type: integer
+        format: int32
+        default: 0
+        minimum: 0
+        description: Offset in resource collection
+      num:
+        type: integer
+        format: int32
+        default: 0
+        minimum: 0
+        maximum: 10000
+        description: Number of resources to retrieve starting at start
+  EntityCollectionResponseV2:
+    type: object
+    properties:
+      href:
+        type: string
+      meta:
+        $ref: "#/definitions/EntityTypeResponseV2"
+      start:
+        type: integer
+        format: int32
+      num:
+        type: integer
+        format: int32
+      total:
+        type: integer
+        format: int64
+      prevHref:
+        type: string
+      nextHref:
+        type: string
+      items:
+        type: array
+        items:
+          type: object
+    required:
+      - href
+      - meta
+      - start
+      - num
+      - total
+      - items
+  EntityTypeResponseV2:
+    type: object
+    properties:
+      href:
+        type: string
+      hrefCollection:
+        type: string
+      name:
+        type: string
+      label:
+        type: string
+      attributes:
+        type: array
+        items:
+          $ref: "#/definitions/AttributeResponseV2"
+      labelAttribute:
+        type: string
+      idAttribute:
+        type: string
+      lookupAttributes:
+        type: array
+        items:
+          type: string
+      isAbstract:
+        type: boolean
+      writable:
+        type: boolean
+      languageCode:
+        type: string
+    required:
+      - href
+      - hrefCollection
+      - name
+      - label
+      - attributes
+      - labelAttribute
+      - idAttribute
+      - lookupAttributes
+      - isAbstract
+      - writable
+      - languageCode
+  AttributeResponseV2:
+    type: object
+    properties:
+      href:
+        type: string
+      fieldType:
+        type: string
+        enum:
+          - BOOL
```

Comment:
Store AttributeType values in model and use in view to prevent swagger ui from breaking when removing/renaming types and automatically add introduced types.